docs/src/modules/styles/getTheme.js

```diff
@@ -12,6 +12,7 @@ function getTheme(uiTheme) {
       background: {
         default: uiTheme.paletteType === 'light' ? '#fff' : '#303030',
       },
+      contrastThreshold: 4.5,
```

Comment:
It's something I have considered doing in the past, but the AppBar colors looks **horrible**. So I never moved forward. What other option to we have?

Reply:
We could change the primary main color. Changing the contrast threshold doesn't solve the root of the problem:

![Capture d’écran 2019-05-19 à 10 58 42](https://user-images.githubusercontent.com/3165635/57984006-1965ee80-7a25-11e9-8572-94b683284777.png)

![Capture d’écran 2019-05-19 à 11 00 02](https://user-images.githubusercontent.com/3165635/57984019-45816f80-7a25-11e9-9360-d177c6fae902.png)